tools-v2/pkg/cli/command/curvebs/status/etcd/etcd.go

```diff
@@ -204,7 +205,7 @@ func NewStatusEtcdCommand() *EtcdCommand {
 	return etcdCmd
 }
 
-func GetEtcdStatus(caller *cobra.Command) (*interface{}, *cmderror.CmdError, cobrautil.ClUSTER_HEALTH_STATUS) {
+func GetEtcdStatus(caller *cobra.Command) (*interface{}, *tablewriter.Table, *cmderror.CmdError, cobrautil.ClUSTER_HEALTH_STATUS) {
```

Comment:
@Cyber-SiKu please take a look this change. It's conflicting with what we discussed earlier.

Reply:
> @Cyber-SiKu please take a look this change. It's conflicting with what we discussed earlier.

Indeed, it seems strange to directly return a table here. I am thinking about whether this is possible? Directly synchronize the status cluster output format to the subcommand instead of simply setting it to noout? Calling this command in this way also has output, but the status cluster only cares about the inspection results of this step?

Reply:
why feel strange? I think that it should be understandable if the output of a command corresponds to a table object. `status cluster` is collection of these tables. The cmd.Result is all rows that seems like need to combine them into a table for output?

Reply:
@Xinlong-Chen @caoxianfei1 
At present, the output of the status cluster is too much, and it is difficult to recognize if the table is directly output. Can you combine the output like this:
services

|role|leader|online|offline|
|---|---|---|---|
|mds|
|etcd|
|snapshot|
|chunkserver|-|

copyset
|total|ok|warn|error|
|---|---|---|---|
|||||

space

|...|...|...|
|---|---|---|
||||

Reply:
Of course, I think both methods are ok.